ui/src/shared/components/TableGraph.js

```diff
@@ -86,14 +92,16 @@ class TableGraph extends Component {
         className={cellClass}
         onMouseOver={this.handleHover(columnIndex, rowIndex)}
       >
-        {`${data[rowIndex][columnIndex]}`}
+        {isFixedColumn
```

Comment:
Determining whether the data is timeseries data by checking whether it is a fixedColumn makes sense here, but since we know the location of timeseries data will vary.. would you mind creating a bool `isTimeData` it's a more understandable name anyway, and for now it can point to isFixedColumn. and whoever changes where timeseries data is can deal with setting it.